cypress/e2e/tests/administration/jira-connection/interlinked.test.ts

```diff
@@ -0,0 +1,152 @@
+        login();
+        jiraBasicCredential = new JiraCredentials(
+            getJiraCredentialData(CredentialType.jiraBasic, useTestingAccount)
+        );
+
+        jiraBasicCredential.create();
+
+        // Defining Jira connection data with correct credentials
+        jiraCloudConnectionData = getJiraConnectionData(
+            jiraBasicCredential,
+            isSecure,
+            useTestingAccount
+        );
+
+        jiraCloudConnection = new Jira(jiraCloudConnectionData);
+
+        // Defining and creating dummy credentials to be used further in tests
+        jiraBasicCredentialInvalid = new JiraCredentials(
+            getJiraCredentialData(CredentialType.jiraBasic, !useTestingAccount)
+        );
+
+        jiraBasicCredentialInvalid.create();
+
+        // Defining Jira connection data with incorrect credentials
+        jiraCloudConnectionDataIncorrect = getJiraConnectionData(
+            jiraBasicCredentialInvalid,
+            isSecure,
+            useTestingAccount
+        );
+
+        jiraCloudConnectionIncorrect = new Jira(jiraCloudConnectionDataIncorrect);
+
+        applicationList = createMultipleApplications(2);
+    });
+
+    beforeEach("Interceptors", function () {
+        cy.intercept("DELETE", "/hub/migrationwaves*/*").as("deleteWave");
+    });
+
+    it("Creating Jira connection with incorrect credentials", () => {
+        jiraCloudConnectionIncorrect.create();
+        jiraCloudConnectionIncorrect.validateState(expectedToFail);
+    });
+
+    it("Bug MTA-1014 | Trying to remove Jira connection used by wave", () => {
+        const issueType = JiraIssueTypes.task;
+        let projectName = "";
+        jiraCloudConnection.create();
+        jiraCloudConnection.validateState(!expectedToFail);
+
+        // Defining and creating new migration wave
+        migrationWave = new MigrationWave(
+            data.getRandomWord(8),
+            now,
+            end,
+            null,
+            null,
+            applicationList
+        );
+        migrationWave.create();
+
+        jiraCloudConnection
+            .getAllProjects()
+            .then((projects) => {
+                expect(!!projects[0]).to.eq(true);
+                projectName = projects[0].name;
+                return jiraCloudConnection.getIssueType(issueType);
+            })
+            .then((issue) => {
+                expect(!!issue).to.eq(true);
+                migrationWave.exportToIssueManager(
+                    JiraType.cloud,
+                    jiraCloudConnection.name,
+                    projectName,
+                    issue.untranslatedName
+                );
+            });
+        Jira.openList();
+        performRowAction(jiraCloudConnection.name, deleteAction);
+        validateTextPresence(jiraAlert, cantDeleteJiraAlert);
+        exists(jiraCloudConnection.name, jiraTable);
+    });
+
+    after("Delete Wave, Jira connection and Jira credentials", () => {
```

Comment:
Let's keep it standard.

after("Clean up data", () => {

Reply:
Done